src/main/java/dk/frankbille/scoreboard/ratings/RatingProvider.java

```diff
@@ -18,13 +18,27 @@
 
 package dk.frankbille.scoreboard.ratings;
 
+import dk.frankbille.scoreboard.ScoreBoardSession;
+import dk.frankbille.scoreboard.domain.RatingCalculatorType;
+import dk.frankbille.scoreboard.domain.User;
 import dk.frankbille.scoreboard.ratings.elo.ELORatingCalculator;
+import dk.frankbille.scoreboard.ratings.trueskill.TrueSkillRatingCalculator;
 
 public class RatingProvider {
 	public static RatingCalculator ratings;
 	public static RatingCalculator getRatings() {
-		if (ratings==null)
-			ratings = new ELORatingCalculator();
+		RatingCalculatorType ratingCalculator = RatingCalculatorType.ELO;
+		if (ScoreBoardSession.get().isAuthenticated()) {
+			User user = ScoreBoardSession.get().getUser();
+			ratingCalculator = user.getRatingCalculator();
+		}
+		if (ratings==null || ratings.getType() != ratingCalculator) {
```

Comment:
Maybe talk to @MKringelhede about it. He introduced it in the first place :smiley: Maybe it just shouldn't be static